src/_layouts/page.html

```diff
@@ -79,8 +79,7 @@
     <link rel="stylesheet" type="text/css" href="/css/main.css" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:ital,wght@0,400;0,500;0,600;0,700;0,800;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet" />
-
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Sue+Ellen+Francisco&display=swap" rel="stylesheet" />
```

Comment:
I haven't used google fonts for a while, but does it really take 3 different `link` tags to get it? 
Can you specify desired weights to shrink the font filesize being sent over the wire?

Reply:
Yes, 3 link tags. Yep, you can choose specific font weights. It doesn't look like it transfers heaps though, just a few KB?